style.css

```diff
@@ -0,0 +1,421 @@
+    background-color: rgb(29,155,240);
+    border-radius: 50px;
+}
+
+#tweet {
+    color: white;
+    font-size: 1rem;
+    padding: 1rem 6rem;
+}
+
+#tweet-button:hover {
+    background-color: rgb(24, 120, 216);
+}
+
+#profile-item {
+    display: grid;
+    grid-template-columns: 1fr 2fr 1fr;
+    place-items: center;
+    padding: 5px 0px;
+}
+
+#profile-item:hover {
+    background-color: rgba(211, 211, 211, 0.5);
+    border-radius: 50px;
+}
+
+#profile-item-user {
+    margin-left: 10px;
+    place-items: center;
+
+}
+
+#profile-item-icon {
+    display: grid;
+    place-items: center;
+    margin-right: 30px;
+}
+
+#profile-item-avatar img {
+    border-radius: 50%;
+    margin-left: 10px;
+}
+
+#middle {
+    display: grid;
+    grid-template-rows: 4rem 1fr;
+    background: rgb(239,243,244);
+    border-right: 1px solid rgba(0, 0, 0, 0.1);
+
+}
+
+#top-bar{
+    display: grid;
+    grid-template-columns: 70px 1fr 40px;
+    border-bottom: 1px solid rgba(0, 0, 0, 0.1);
+    background: white;
+}
+
+#page-title-area {
+    height: 4rem;
+    display: grid;
+    place-items: center;
+    padding: 0px 20px;
+}
+
+#page-icon-area {
+    display: grid;
+    place-items: center right;
+    margin-right: 20px;
+}
+
+#status-container {
+    height: 8rem;
+    display: grid;
+    grid-template-columns: auto 1fr;
+    border-bottom: 1px solid rgba(0, 0, 0, 0.1);
+    background: white;
+}
+
+#status-left {
+    height: 8rem;
+    padding: 10px 15px 20px 15px;
+}
+
+#status-left img {
+    border-radius: 50%;
+}
+
+#status-right {
+    height: 8rem;
+    display: grid;
+    grid-template-rows: 4rem 3rem;
+}
+
+#status-right-top {
+    display: grid;
+    place-items: center left;
+}
+
+#status-right-bottom {
+    display: grid;
+    grid-template-columns: 1fr 20%;
+}
+
+#status-right-bottom-svg {
+    fill: rgb(29, 155, 240);
+    display: flex;
+    width: 220px;
+}
+
+#status-right-bottom-svg svg {
+    margin-right: 20px;
+}
+
+#status-right-bottom-tweet {
+    display: grid;
+    background-color: rgba(29, 155, 240, 0.5);
+    border-radius: 50px;
+    place-items: center;
+    margin-right: 10px;
+}
+
+#status-right-bottom-tweet a {
+    color: white;
+    text-decoration: none;
+    font-weight: bold;
+    text-align: center;
+}
+
+#feed {
+    display: grid;
+    grid-auto-rows: auto;
+}
+
+.feed-cards-container {
+    background: white;
+    border-bottom: 1px solid rgb(213, 214, 214);
+    display: grid;
+    grid-template-columns: auto 1fr;
+}
+
+.feed-cards-container img {
+    width: 80px;
+    border-radius: 50%;
+    margin: 20px;
+}
+
+.feed-posts-container {
+    margin-top: 20px;
+    display: grid;
+}
+
+.user-name {
+    display: inline;
+}
+
+.feed-posts-container p.twitter-handle {
+    color: grey;
+    font-size: 1rem;
+}
+
+.feed-posts-container p.twitter-post {
+    display: block;
+    margin: 8px 0px;
+    width: 60ch;
+    line-height: 1.5;
+}
+
+.post-image {
+    border-radius: 4px !important;
```

Comment:
Usually it's a bad idea to use `!important`. Typically when you feel you need to use it, you're often applying the css in the wrong place/on the wrong element. 

The reason why we don't like to use `!important` is because if we ever want to overwrite this style, we will use another `!important`.

Reply:
Yes I know it's not great but I didn't know how to overcome the issue and I thought it was a good idea using it in view of the size of the clone. On bigger sites it would be different.